core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -482,7 +488,11 @@ export class ExecuteWorkflowService {
       ExecuteWorkflowService.transformBreakpoint(workflowGraph, e[0], e[1])
     );
 
-    return { operators, links, breakpoints };
+    const cachedOperatorIDs: string[] = Array.from(
```

Comment:
return readonlyarray?

Reply:
no need to return readonly array, the type definition is ready only, which is enough